pom.xml

```diff
@@ -194,7 +195,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.18.1</version>
                 <configuration>
-                    <argLine>-XX:-UseSplitVerifier</argLine>
+                    <argLine>${argLine} -XX:-UseSplitVerifier</argLine>
```

Comment:
In the coverage PR of osiam/osiam you completely removed the `-XX:-UseSplitVerifier` argument. Why isn't this the case here too?